protostar/testing/cheatcodes/send_message_to_l2.py

```diff
@@ -81,5 +87,6 @@ def send_message_to_l2(
                 entry_point_type=EntryPointType.L1_HANDLER,
                 call_type=CallType.DELEGATE,
                 class_hash=class_hash,
+                initial_gas=10**10,
```

Comment:
It appears in another place

Reply:
[Use constant from cairo-lang instead of coming up with a number](https://github.com/software-mansion/protostar/pull/1526/commits/250c3bd0cd85b2ab0f41c7541aa46e7d91fe7bf4)